schema.sql

```diff
@@ -0,0 +1,23 @@
+DROP DATABASE IF EXISTS sweethornbillbookstore;
+CREATE DATABASE sweethornbillbookstore;
+
+\c sweethornbillbookstore;
+
+
+CREATE TABLE books (
+  id SERIAL PRIMARY KEY,
+  title VARCHAR,
+  author VARCHAR,
+  genre VARCHAR,
+  description VARCHAR,
+  img_url VARCHAR
+);
```

Comment:
VARCHARs should have length limits on them to prevent unnecessary storing of too much data. `title VARCHAR(100)`